src/labelle/gui/gui.py

```diff
@@ -63,7 +63,7 @@ def _init_elements(self) -> None:
         self._device_manager = DeviceManager()
         self._dymo_labeler = DymoLabeler()
         self._settings_toolbar.update_labeler_context(
-            supported_tape_sizes=self._dymo_labeler.labeler_supported_tape_sizes,
+            supported_tape_sizes=self._dymo_labeler._deviceConfig.supportedTapeSizes,
```

Comment:
Maybe instead of exposing inner `_deviceConfig`, add property to the labeler class to which exposes it.

Reply:
You mean with the `@property` thing? I'll give it a shot (sorry python noob here)

Reply:
The `@property` thing is called a "decorator". That's a bit of a rabbit-hole...

Python's property mechanism is a way to spoof function calls to appear as attributes. They are popular, but also considered by some to be hacky.

I think it's fine for now if you just define a getter function.

Reply:
Fixed (I think)